packages/core/src/checkout/checkout-service.ts

```diff
@@ -1443,6 +1444,17 @@ export default class CheckoutService {
         this._extensionMessenger.clearCacheByRegion(region);
     }
 
+    /**
+     * Posts a message to a checkout extension.
+     *
+     * @alpha
+     * @param extensionId - The ID of an extension to post the event to.
+     * @param message -The message to post to an extension.
+     */
+    postMessageToExtension(extensionId: string, message: ExtensionMessage): void {
```

Comment:
How come we need to expose this now?

Reply:
Previously, we only had a broadcaster that delivers messages from checkout-js to an extension, with the messenger wrapped inside the broadcaster. 
https://github.com/bigcommerce/checkout-sdk-js/blob/6a9b5171d8c000a7fe43702f906306f4b0e50f7b/packages/core/src/extension/extension-event-broadcaster.ts#L56
Exposing this method now allows a checkout-js command handler to send messages.